Samples/VirtualCamera/VirtualCamera_ConsoleApp/readme.txt

```diff
@@ -0,0 +1,122 @@
+========================================================================
+About the Project
+========================================================================
+Basic console application that validate virtual camera source implementation, 
+register as virtual camera, remove virtual camera, validate virtual camera works 
+thru mediacapture API
```

Comment:
orth: "validate**s**", "register**s**", "remove**s**". "thru" should be "through"

Reply:
i have removed project level readme(s) and just keep one at the solution level